### DISCUSSION TITLE
How to use vactubes with angles?

### DISCUSSION THREAD

**Discussion by revaru** [BEEmod/BEE2-items#4200] [Tech Support / Q&A] [answered]
In BEEmod when i connecting vactubes, it don't connecting with angles,
I may demonstrate to you.
Imagine:
- = void
VACTUBE = Vactube (Logic)
) = Turn
If i connecting like that:
VACTUBE - - - - - VACTUBE
This is just straight, and
VACTUBE - - - - ) - - - - VACTUBE
It is don't connecting.
HOW TO DO ANGLES?! If you don't understand what i writing, just explain to me how to do angles in vactubes.

**Comment by CTB33391** [comment]
Vactube corners can only be between 1x1 voxels and 3x3 voxels/ The spacing on your diagrams appears to be 4 voxels. Bring them closer.
Also ensure they are facing the right directions. They must not be pointing towards eachother like this
.>..
....^
In that case - assuming you want a downward corner - you change it to
.>..
.....v
Hope that helps.

**Comment by Endermage77** [comment]
Generally, you need another tube item every time you make a turn with the tubes.

Pointing in the right direction, of course.

**Comment by revaru** [comment]
I want to add that I do not understand the principle they give different sizes vactubes, ie 1x1, 2x2, 3x3

**Comment by CubieJudy** [comment]
https://github.com/BEEmod/BEE2-items/wiki/Vactubes

**Comment by HugoBDesigner** [comment, marked as answer]
You can have straight tubes or 90° curves of 1x1, 2x2 or 3x3. You cannot have twisted, tilted or offset vactubes. See link above for more information on usage.

**Comment by revaru** [comment]
Thanks very much
